cdk/visit/console/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
```

Comment:
Are your changes to `cdk/visit/console/` still necessary now that https://github.com/clemsonMakerspace/unified-makerspace/pull/92 has been merged? I think that the pipeline should automatically build the site and we don't need to check in any artifacts now.